test/test_masks_to_boxes.py

```diff
@@ -0,0 +1,34 @@
+import os.path
+
+import PIL.Image
+import numpy
+import torch
+
+from torchvision.ops import masks_to_boxes
+
+ASSETS_DIRECTORY = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
+
+
+def test_masks_to_boxes():
+    with PIL.Image.open(os.path.join(ASSETS_DIRECTORY, "masks.tiff")) as image:
+        frames = numpy.zeros((image.n_frames, image.height, image.width), int)
```

Comment:
Same here

Reply:
@0x00b1 Could make the same changes you did on the galleries also here?
https://github.com/pytorch/vision/pull/4290/files#diff-26925685a7a896911607f04c21b03d9b2c029a0a9530c7006f01c1a994ab8448R45-R53

Aka replace numpy with torch.